# Description: restore getter function needed by some other patch,
# should let things build while we refresh to the other changesets
# not to forward upstream in any case
Index: nautilus-3.18.1/src/nautilus-list-view.c
===================================================================
--- nautilus-3.18.1.orig/src/nautilus-list-view.c
+++ nautilus-3.18.1/src/nautilus-list-view.c
@@ -3223,3 +3223,9 @@ nautilus_list_view_new (NautilusWindowSl
 			     "window-slot", slot,
 			     NULL);
 }
+
+GtkTreeView*
+nautilus_list_view_get_tree_view (NautilusListView *list_view)
+{
+	return list_view->details->tree_view;
+}
Index: nautilus-3.18.1/src/nautilus-list-view.h
===================================================================
--- nautilus-3.18.1.orig/src/nautilus-list-view.h
+++ nautilus-3.18.1/src/nautilus-list-view.h
@@ -53,5 +53,6 @@ typedef struct {
 
 GType nautilus_list_view_get_type (void);
 NautilusFilesView * nautilus_list_view_new (NautilusWindowSlot *slot);
+GtkTreeView* nautilus_list_view_get_tree_view (NautilusListView *list_view);
 
 #endif /* NAUTILUS_LIST_VIEW_H */
